package.json

```diff
@@ -38,7 +38,7 @@
         "@azure/core-http": "^1.2.1-alpha.20201203.2",
```

Comment:
@joheredi @ramya-rao-a Is it Ok to release a preview version of SDK generator with alpha version of core-http? I think this also should be updated to the latest version. Any thoughts???

Reply:
this is a pretty old version from last year. I think we should update it to a GA one.

Reply:
Yes, we should change this to ^1.2.4

Reply:
Done